app/lib/actions/foreman/host/import_facts.rb

```diff
@@ -8,7 +8,7 @@ def resource_locks
 
         def plan(_host_type, host_name, facts, certname, proxy_id)
           facts['domain'].try(:downcase!)
-          host = ::Host::Base.import_host(host_name, certname, facts, proxy_id)
+          host = ::Host::Base.import_host(host_name, certname, proxy_id)
```

Comment:
This PR actually _depends_ on https://github.com/theforeman/foreman/pull/3649. Sorry about the misunderstanding. I am changing the signature of `#import_host` method [there](https://github.com/theforeman/foreman/pull/3649/files#diff-07cbb99bfdb4bed86ad02486bb5adde2R106).